sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

```diff
@@ -8,8 +8,11 @@
 
 ### Bugs Fixed
 
+- Fixed [#5297](https://github.com/Azure/azure-sdk-for-cpp/issues/5297). The actual fix for this was to use a single management client per connection.
```

Comment:
Remove "Fixed", make it in the format of something like`- [#1234] Management API to use a separate session.`